### PR TITLE
feature: use contextal logging 

### PR DESCRIPTION
**Pull request by googs1025**
#### What this PR does / why we need it:
- use contextal logging

Due to the large scope, I may only be able to complete part of the plugins work in this PR.
- coscheduling
- capacityscheduling
- networkaware
- sysched
#### Which issue(s) this PR fixes:

Fixes # part of https://github.com/kubernetes-sigs/scheduler-plugins/issues/743

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: pkg/coscheduling/core/core.go [hunk 1]
```diff
@@ -211,6 +213,8 @@ func (pgMgr *PodGroupManager) PreFilter(ctx context.Context, pod *corev1.Pod) er
 
 // Permit permits a pod to run, if the minMember match, it would send a signal to chan.
 func (pgMgr *PodGroupManager) Permit(ctx context.Context, state *framework.CycleState, pod *corev1.Pod) Status {
+	lh := klog.FromContext(ctx)
```
Comment (googs1025) [inline pkg/coscheduling/core/core.go:216]: `klog.FromContext(ctx)` 
When using klog.FromContext(ctx), do I need to add WithValues ​​to give it more information? 🤔

`klog.FromContext(ctx).WithValues("PodGroupManager", "Unreserve")`

File: pkg/capacityscheduling/capacity_scheduling.go [hunk 2]
```diff
@@ -125,6 +125,7 @@ func New(ctx context.Context, obj runtime.Object, handle framework.Handle) (fram
 		podLister:         handle.SharedInformerFactory().Core().V1().Pods().Lister(),
 		pdbLister:         getPDBLister(handle.SharedInformerFactory()),
 	}
+	logger := klog.FromContext(ctx).WithName(Name).WithValues("Phase", "New")
```
Comment (Huang-Wei) [inline pkg/capacityscheduling/capacity_scheduling.go:128]: To ease reviewing, I'm inclined to keep this PR merely as a "migration" PR - i.e., keep new bits in a follow-up, only migrate klog.InfoS / klog.ErrorS.

File: pkg/capacityscheduling/capacity_scheduling.go [hunk 3]
```diff
@@ -480,25 +491,27 @@ func (p *preemptor) SelectVictimsOnNode(
 	pod *v1.Pod,
 	nodeInfo *framework.NodeInfo,
 	pdbs []*policy.PodDisruptionBudget) ([]*v1.Pod, int, *framework.Status) {
+
+	logger := klog.FromContext(ctx).WithName(Name).WithValues("Phase", "SelectVictimsOnNode")
```
Comment (Huang-Wei) [inline pkg/capacityscheduling/capacity_scheduling.go:495]: ditto. Would you mind folding the "changes" in a follow-up PR?

Comment (googs1025) [reply pkg/capacityscheduling/capacity_scheduling.go:495]: ok, I will revert this

Comment (googs1025) [reply pkg/capacityscheduling/capacity_scheduling.go]: done

File: pkg/capacityscheduling/capacity_scheduling.go (hunk 2 above)
Comment (googs1025) [reply pkg/capacityscheduling/capacity_scheduling.go]: done

**Comment by googs1025** [comment]
> Thanks for the PR @googs1025 !
> 
> One question: do you plan to work on some script to enforce contextual logging into CI? (I may assume we may leverage from k/k)

Ok, this is something I need to look into, I'm not familiar with this part of the script. If you could point me to some help, that would be great :)

**Review by Huang-Wei — COMMENTED** [review]
/lgtm
/approve

**Comment by Huang-Wei** [comment]
When migrating to structural logging, we composed a script hack/verify-structured-logging.sh to enforce it - which was based on k8s.io/klog/hack/tools/logcheck.

For contextual logging, we may proceed with a similar approach.

**Pull request merged**